### PR TITLE
Replace SCSS modules styling with Tailwind

### LINKED ISSUE #15: Replace SCSS modules with Tailwind
To have a more consistent tech stack across the AlveusGG projects, look to replace the SCSS modules usage here with Tailwind.

Reference the tailwind config in https://github.com/alveusgg/alveusgg/blob/main/apps/website/tailwind.config.ts, and perhaps think about how we could even abstract that into a shared package (like alveusgg/data), so our theme is consistent.

### PR DESCRIPTION
**Pull request by MattIPv4**
Resolves #15

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,7 +22,8 @@
     "format": "run-p format:*"
   },
   "dependencies": {
-    "@alveusgg/data": "github:alveusgg/data#0.39.0",
+    "@alveusgg/data": "github:alveusgg/data#MattIPv4/tailwind-extras",
```
Comment (MattIPv4) [inline package.json:25]: Switch to released version

File: src/components/tooltip/Tooltip.tsx [hunk 2]
```diff
@@ -96,14 +96,19 @@ const Tooltip = (props: TooltipProps) => {
     <>
       {childrenWithProps}
       <div
-        className={styles.tooltip}
+        className="bg-black/50 shadow-lg backdrop-blur rounded-lg p-2 pointer-events-none fixed w-max z-10 transition-opacity "
```
Comment (MattIPv4) [inline src/components/tooltip/Tooltip.tsx:99]:
```suggestion
        className="bg-black/50 shadow-lg backdrop-blur rounded-lg p-2 pointer-events-none fixed w-max z-10 transition-opacity"
```

File: src/pages/overlay/components/overlay/settings/Settings.tsx [hunk 3]
```diff
@@ -6,21 +6,23 @@ import Card from "../../../../../components/card/Card";
 import Toggle from "../../toggle/Toggle";
 
 import type { OverlayOptionProps } from "../Overlay";
-
-import styles from "./settings.module.scss";
+import { classes } from "../../../../../utils/classes";
```
Comment (MattIPv4) [inline src/pages/overlay/components/overlay/settings/Settings.tsx:9]: Sort import

**Review by MattIPv4 — COMMENTED** [review]
Prettier class sorting?

File: src/pages/overlay/components/overlay/ambassadors/Ambassadors.tsx [hunk 4]
```diff
@@ -84,58 +92,74 @@ export default function Ambassadors(props: OverlayOptionProps) {
...
-          className={styles.ambassadorCard}
-        />
-      )}
+      {ambassadors.map(([key]) => (
+        <Transition show={activeAmbassador.key === key} key={key}>
+          <AmbassadorCard
+            key={key}
+            ambassador={key}
+            onClose={() => setActiveAmbassador({})}
+            className="z-0 col-start-2 row-start-1 origin-[center_left] self-center transition-[opacity,transform] data-[closed]:-translate-x-10 data-[closed]:opacity-0 data-[closed]:motion-reduce:translate-x-0"
```
Comment (pjeweb) [inline src/pages/overlay/components/overlay/ambassadors/Ambassadors.tsx:159]: Chrome and Firefox are a bit jumpy without `will-change` for me shifting the text and card height a tiny bid after the transition. I think this fixed that.

Also I'd probably up the transition duration a bid. Feels a lot smoother to me.

```suggestion
            className="z-0 col-start-2 row-start-1 origin-[center_left] self-center transition-[opacity,transform] data-[closed]:-translate-x-10 data-[closed]:opacity-0 data-[closed]:motion-reduce:translate-x-0 duration-500 will-change-transform"
```

**Review by pjeweb — APPROVED** [review]
Very nice

File: src/pages/overlay/components/overlay/ambassadors/Ambassadors.tsx (hunk 4 above)
Comment (pjeweb) [reply src/pages/overlay/components/overlay/ambassadors/Ambassadors.tsx:159]: Maybe not 500ms. I have not checked what the other durations are set to.

**Comment by pjeweb** [comment]
Hmm and the IUCN tooltip does not sit right for me.

File: src/pages/overlay/components/overlay/ambassadors/Ambassadors.tsx (hunk 4 above)
Comment (MattIPv4) [reply src/pages/overlay/components/overlay/ambassadors/Ambassadors.tsx:159]: Ah yeh, the old transitions globally were 300ms, Tailwind will default to 150ms. Will bump that back up.

**Comment by MattIPv4** [comment]
> Hmm and the IUCN tooltip does not sit right for me.

Oh, how so?

**Comment by pjeweb** [comment]
> > Hmm and the IUCN tooltip does not sit right for me.
> 
> Oh, how so?

Apparently thats a side effect of will-change ...

**Comment by MattIPv4** [comment]
Ah yeh, setting will-change creates a new stacking context, which'll change how absolute positioning works

**Review by pjeweb — APPROVED** [review]

**Pull request merged**